bin/ruby-build

```diff
@@ -701,7 +701,7 @@ fix_rbx_irb() {
 }
 
 require_java7() {
-  local version="$(java -version 2>&1 | grep 'java version' | head -1)"
+  local version="$(java -version 2>&1 | grep '\(java\|openjdk\) version' | head -1)"
```

Comment:
This doesn’t work on OS X (BSD grep). Grep command should be replaced with: `grep -E '(java|openjdk) version'`. This works with both BSD and GNU grep.